client/components/mma/cancel/cancellationSaves/SelectReason.tsx

```diff
@@ -60,7 +62,7 @@ const CancellationInfo = ({
 		`}
 	>
 		<Stack space={1}>
-			<p css={paragraphListCss}>
+			<p css={paragraphListCss} data-qm-masking="blocklist">
```

Comment:
I think there's another email on the switch complete page

Reply:
This is already blocked from when we implemented the Switch journey

Reply:
ah yeah nice 👍